### ISSUE TITLE
[Bug] Can not see the content of added files in "Compare commit" view.

### ISSUE THREAD

**Issue by vicb** [DonJayamanne/gitHistoryVSCode#570]
In the "compare commits" view:
- Added files are displayed prefixed with a "A" icon,
- Modified files are displayed prefixed with a "M" icon.

Clicking on a "M" file would display a diff view.

Clicking on a "A" files does nothing. It would be great to open that file for review.

And Don, thank you so much for this helpful extension. It really helps me being more productive. Great job !

Thanks

**Comment by ole1986** [comment]
When you click on a file at least the context should appear where you are allowed to compare against workspace file. Does this work for you?

Is the context menu opening for "Added" files?

**Comment by vicb** [comment]
> When you click on a file at least the context should appear where you are allowed to compare against workspace file. Does this work for you?

If you mean a right click, yes this works today.

I was actually referring to a left click on the file.
Before when I used to left click on an added file, it would not open in the editor.

It seems to work now - I am using 0.6.10 (Sorry I forgot to mention the version where I had the bug in my initial bug report).

So it seems like my problem is solved. Closing this issue.

Thanks for this great plugin !
